Sources/Controllers/Map/OAMapInfoController.mm

```diff
@@ -408,6 +411,25 @@ - (void) layoutWidgets:(OATextInfoWidget *)widget
         CGRect f = _rightWidgetsView.superview.frame;
         _rightWidgetsView.superview.frame = CGRectMake(f.origin.x, f.origin.y, f.size.width, maxContainerHeight);
     }
+    
+    if (_topCoordinatesView && _topCoordinatesView.superview && !_topCoordinatesView.hidden)
+    {
+        if (_lastUpdateTime == 0)
+            [[OARootViewController instance].mapPanel updateToolbar];
+        
+        BOOL isMarkerVidgetVisible = [_mapHudViewController getControlsTopPosition] > 0;
```

Comment:
Looks like wrong name. Is getControlsTopPosition related to Marker only?
Vidget > Widget